server/db/users.js

```diff
@@ -1,3 +1,34 @@
+const environment = process.env.NODE_ENV || 'development'
+const config = require('./knexfile')[environment]
+const knex = require('knex')(config)
```

Comment:
Let's call this variable `connection` instead of `knex` so it's consistent with the other database modules. This change will force you to edit the input parameters of the functions below.